executor/internal/calibrateresource/calibrate_resource.go

```diff
@@ -252,29 +255,46 @@ func (e *Executor) dynamicCalibrate(ctx context.Context, req *chunk.Chunk, exec
 	if err != nil {
 		return err
 	}
+	tidbQuota, err1 := e.getTiDBQuota(ctx, exec, startTs, endTs)
+	tiflashQuota, err2 := e.getTiFlashQuota(ctx, exec, startTs, endTs)
+	if err1 != nil && err2 != nil {
+		return err1
+	}
+	if err1 != nil {
+		logutil.BgLogger().Error("get tidb/tikv ru quota failed", zap.Error(err1))
+	}
+	if err2 != nil {
+		logutil.BgLogger().Error("get tiflash ru quota failed", zap.Error(err2))
+	}
+	req.AppendUint64(0, uint64(tidbQuota+tiflashQuota))
+	return nil
+}
+
+func (e *Executor) getTiDBQuota(ctx context.Context, exec sqlexec.RestrictedSQLExecutor, startTs, endTs time.Time) (float64, error) {
```

Comment:
Seems like it includes TiKV Quota, does it make sense to be called `getTiDBQuota`?

Reply:
maybe `getTiDBAndTiKVQuota` ?